charts/kubeclarity/values.yaml

```diff
@@ -302,11 +302,6 @@ kubeclarity-postgresql:
     # user:
     # password:
 
-  ## Setup database name and password
-  auth:
-    existingSecret: kubeclarity-postgresql-secret
-    database: kubeclarity
```

Comment:
Everything under kubeclarity-postgresql are part of the value schema from the postgres sub-chart, these can not be removed from here otherwise kubeclarity-postgresql will not be configured properly.

I think we need to have a new section that isn't reuse for remote postgres, and then template based on kubeclarity-postgresql being enabled or remote being enabled.

Reply:
I'll do that. Thanks for your advice.

Reply:
I put back `kubeclarity-postgresql` to normal and added user and port configuration there.

Reply:
In the future please allow the original commenter to resolve the conversations so they can determine if their comment was resolved or not.